bin/build-branch.php

```diff
@@ -29,8 +29,20 @@ function getLicense()
 EOT;
 }
 
+public function minPhpVersionFor(string $version): string
```

Comment:
```suggestion
function minPhpVersionFor(string $version): string
```

oops sorry missed this

Reply:
done